spec/features/view_scheduled_messages_spec.rb

```diff
@@ -44,24 +44,44 @@
     expect(page).to have_content(second_scheduled_message.title)
     expect(page).to have_content(second_scheduled_message.body)
     expect(page).to have_content(second_scheduled_message.days_after_start)
-    expect(page).to have_content(second_scheduled_message.time_of_day.strftime("%l:%M %p"))
+    expect(page).to have_content(get_formatted_time(second_scheduled_message))
 
     expect(page).to have_content("Prev")
     expect(page).to have_content("First")
   end
 
+  scenario "see message in the correct order" do
+    login_with_oauth
+    visit root_path
+    create_scheduled_messages
+
+    visit scheduled_messages_path
+
+    expect(page).to have_selector("table > tr:nth-child(2) > td:nth-child(2)",
+                                  text: get_formatted_time(first_scheduled_message).strip)
+    expect(page).to have_selector("table > tr:nth-child(3) > td:nth-child(2)",
+                                  text: get_formatted_time(second_scheduled_message).strip)
+  end
+
   private
 
+  def get_formatted_time(scheduled_message)
```

Comment:
what about naming this `formatted_time` instead of `get_formatted_time` ? seems like it is as clear and shorter!